superchain/configs/configs.json

```diff
@@ -151,6 +151,68 @@
             "UnsafeBlockSigner": "0xD1705B4FFFc540EDeD73046ee1F3A8Db10d143f8"
           }
         },
+        {
+          "Name": "Swan Chain Mainnet",
+          "l2_chain_id": 254,
+          "PublicRPC": "https://mainnet-rpc.swanchain.org",
+          "SequencerRPC": "https://sequencer-mainnet.swanchain.org",
+          "Explorer": "https://swanscan.io",
+          "SuperchainLevel": 0,
+          "StandardChainCandidate": false,
```

Comment:
I have in our internal notes that Swan Chain is supposed to be a standard chain, are our notes incorrect?